diff --git a/Makefile b/Makefile
index 561fb0d..859485a 100644
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ CPPFLAGS = -Ilua/src -Isrc -Iunittest/googletest/googletest/include -Iunittest/g
 CXXFLAGS ?= $(CFLAGS) -fno-exceptions
 
 CXXLIBFLAGS ?=
-LDFLAGS += -L$(BUILDDIR) -ltundra
+LDFLAGS += -L$(BUILDDIR) -ltundra -lgtest
 
 PREFIX ?= /usr/local
 
diff --git a/unittest/TestHarness.cpp b/unittest/TestHarness.cpp
index 8f7cb95..a86ab0a 100644
--- a/unittest/TestHarness.cpp
+++ b/unittest/TestHarness.cpp
@@ -1,10 +1,8 @@
 #include "TestHarness.hpp"
 #include "gtest/gtest.h"
-#include "googletest/googletest/src/gtest-all.cc"
 
-int main(int argc, char* argv[])
-{
+
+int main(int argc, char **argv) {
   ::testing::InitGoogleTest(&argc, argv);
   return RUN_ALL_TESTS();
 }
-

